src/textbooks/textbook-card/TextbooksCard.test.jsx

```diff
@@ -1,31 +1,65 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import MockAdapter from 'axios-mock-adapter';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { initializeMockApp } from '@edx/frontend-platform';
 
-import userEvent from '@testing-library/user-event';
-import TextbookCard from './TextbooksCard';
+import messages from '../textbook-form/messages';
+import { getEditTextbooksApiUrl } from '../data/api';
+import { editTextbookQuery } from '../data/thunk';
 import { textbooksMock } from '../__mocks__';
+import initializeStore from '../../store';
+import { executeThunk } from '../../utils';
+import TextbookCard from './TextbooksCard';
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
[question]: Do we need this unnecessary eslint disable?

Reply:
Fixed